reports/Colorado.md

```diff
@@ -6,15 +6,16 @@ On June 27th, citizens of Aurora held a violin vigil in honor of Elijah McClain.
 
 Aurora police arrived in riot gear to disperse the vigil. When participants refused, riot police beat them with batons and pepper-sprayed them.
 
-tags: strike, beat, baton, pepper-spray, spray
+tags: strike, beat, baton, pepper-spray, spray, tear-gas
 
 id: co-aurora-1
 
 **Links**
 
 * https://www.reddit.com/r/Bad_Cop_No_Donut/comments/hhfrh8/062820_musicians_in_aurora_colorado_honoring/
 * https://www.reddit.com/r/2020PoliceBrutality/comments/hhh6fe/aurora_co_during_a_vigil_with_live_violinists_for/
-* https://twitter.com/jessiedesigngir/status/1277126019246260224
```

Comment:
Why did you remove this link? It shows the police arriving on scene

Reply:
I thought that link was already in one of the above reddit posts, but I was mistaken

Reply:
Added link back